lucille-examples/lucille-s3-ingest-example/conf/s3-opensearch.conf

```diff
@@ -0,0 +1,73 @@
+connectors: [
+ {
+    name: "fileConnector",
+    class: "com.kmwllc.lucille.connector.FileConnector",
+    pipeline: "pipeline1",
+    pathToStorage: ${PATH_TO_STORAGE}
+    cloudOptions: {
+      region: ${AWS_REGION}
+      accessKeyId: ${AWS_ACCESS_KEY_ID}
+      secretAccessKey: ${AWS_SECRET_ACCESS_KEY}
+    }     
+ }
+]
+
+pipelines: [
+  {
+    name: "pipeline1",
+    stages: [
+     {
+       name:"TextExtractor"
+       class: "com.kmwllc.lucille.tika.stage.TextExtractor"
+       byte_array_field: "file_content"
+       metadata_prefix: ""
+       metadata_blacklist: []
+       tika_config_path: "conf/tika-config.xml"       
+     },
+     {
+      name:"ChunkText"
+      class: "com.kmwllc.lucille.stage.ChunkText"
+      source: "text"
+      chunking_method: "sentence"
+      # how many smaller chunks to merge into one final chunk
+      # chunks_to_merge: 2
+      # chunks_to_overlap: 1
+     },
+     {
+      name: "EmitNestedChildren"
+      class : "com.kmwllc.lucille.stage.EmitNestedChildren"
+      drop_parent: "true"
+     },
+     {
+      name: "DeleteFields"
+      class : "com.kmwllc.lucille.stage.DeleteFields"
+      fields : ["file_content", "_version"]
+     },
+      {
```

Comment:
Suggest commenting out this Print stage.
It's useful for for debugging but not needed for the end-to-end example.